pylace/lace/engine.py

```diff
@@ -201,21 +203,19 @@ def columns(self):
 
     @property
     def index(self):
-        """
-        The string row names of the engine
-        """
+        """The string row names of the engine."""
         return self.engine.index
 
     @property
     def ftypes(self):
         """
-        A dictionary mapping column names to feature types
+        A dictionary mapping column names to feature types.
 
         Examples
         --------
         >>> from lace.examples import Satellites  # doctest: +SKIP
-        >>> engine = Satellites()                 # doctest: +SKIP
-        >>> engine.ftypes                         # doctest: +SKIP
+        >>> engine = Satellites()  # doctest: +SKIP
```

Comment:
Personally I disagree with this, which I gather comes from `blackdoc`? But I'm not gonna bikeshed over it.